homeassistant/components/tomorrowio/const.py

```diff
@@ -32,6 +33,7 @@
 DEFAULT_NAME = INTEGRATION_NAME
 ATTRIBUTION = "Powered by Tomorrow.io"
 
+DEFAULT_MAX_REQUESTS_PER_DAY = 500
```

Comment:
The default should be 100 so that it just works for new API tokens.

Reply:
For existing installations it should load the already configured value from the config store, no?

Reply:
The low-level API (pytomorrowio project) has `max_requests_per_day` method to obtain contents of the rate limit HTTP header (when present). Otherwise it defaults to 100.

So I *think* every free API token is currently limited to 100 API calls per day. And that should be the default when the config entry is missing. It would be ideal to have an OptionsFlow to allow existing users to change the limit without having to remove and re-add the integration.